src/create_features.py

```diff
@@ -170,6 +164,11 @@ def create_features(comments_df, training, G):
   comments_df = convo_politeness.get_politeness_score(
       comments_df)
 
+  # try sentiment
```

Comment:
?

Reply:
oh! in the middle of adding sentiment analysis.. I'll remove it for this branch and put it in a new branch